### PR TITLE
RF | Creating headerSpec file

### PR DESCRIPTION
**Pull request by ElviraAvshalumov**
https://trello.com/c/RgRHloYx/841-rf-creating-headerspec-file

### CODE REVIEW

File: cypress/e2e/headerSpec.cy.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+const guidePage = new GuidePage();
+const header = new Header();
+
+describe('Header test suit', () => {
+
+    beforeEach(function () {
+        cy.fixture('asiaJS').then(data => {
+            this.data = data
+        });
```
Comment (Rom-T) [inline cypress/e2e/headerSpec.cy.js:15]: this fixture is not used anywhere

**Review by Rom-T — APPROVED** [review]

File: cypress/e2e/headerSpec.cy.js (hunk 1 above)
Comment (ElviraAvshalumov) [reply cypress/e2e/headerSpec.cy.js]: fixed

**Review by StanislavBliss — APPROVED** [review]

**Comment by ElviraAvshalumov** [comment]
[RF | Creating headerSpec file](https://trello.com/c/RgRHloYx/841-rf-creating-headerspec-file)

**Review by Rom-T — DISMISSED** [review]

**Review by kgoldpasv — APPROVED** [review]
ok

**Review by Rom-T — APPROVED** [review]

**Review by MarinaMone — APPROVED** [review]

**Review by qa-tester22 — APPROVED** [review]

**Pull request merged**